src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -331,7 +337,8 @@ public LineGeometry3D Axes
 
         public static PhongMaterial SelectedMaterial { get; set; }
 
-        public static PhongMaterial TransparentMaterial { get; set; }
```

Comment:
Forgot if we agreed that we can remove public properties in this branch?

Reply:
this was added by @aparajit-pratap - it's not an existing member in 2.5 (I don't think)